src/app/dataset-view/dataset.component.ts

```diff
@@ -63,9 +77,34 @@ export class DatasetComponent extends BaseDatasetDataComponent implements OnInit
             _.isNil(this.datasetBasics) ||
             this.datasetBasics.name !== urlDatasetInfo.datasetName ||
             this.datasetBasics.owner.accountName !== urlDatasetInfo.accountName ||
-            this.datasetViewType === DatasetViewTypeEnum.Flows
+            [DatasetViewTypeEnum.Overview, DatasetViewTypeEnum.Metadata].includes(this.datasetViewType)
```

Comment:
Why only these tabs? When I select `Data` tab, I want to see relevant schema too.

Reply:
Added `Data` tab